embulk-jruby-strptime/src/main/java/org/embulk/spi/time/RubyDateParser.java

```diff
@@ -0,0 +1,112 @@
+package org.embulk.spi.time;
+
+import org.jruby.runtime.ThreadContext;
```

Comment:
See my comment for `StrptimeParser.java`. It means that we may want to remove only the dependency here without removing our own implementation in Embulk.